core/migrations/0014_auto_20200507_1756.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.0.6 on 2020-05-07 17:56
```

Comment:
If you rebase or merge master, you shouldn't have any migrations or changes to models or tasks.